src/BuildScriptGenerator/Ruby/RubyBashBuildSnippet.sh.tpl

```diff
@@ -0,0 +1,17 @@
+echo
+echo "Using Gem version:"
+gem --version
+echo
+echo
+echo "Using Ruby version:"
+ruby --version
+echo
+
+{{ if UseBundlerToInstallDependencies }}
+echo "Running 'gem install bundler'..."
```

Comment:
Yes, ruby installation will come with gem if ruby version is newer than 1.*, since we have 2.* so we're good. A Gem is a form of package and if we found a Gemfile, then use bundler to install all dependencies inside the Gemfile. I Will think of the case when there has no Gemfile included in the app.